src/bloaty.h

```diff
@@ -20,6 +20,7 @@
 #define BLOATY_H_
 
 #include <stdlib.h>
+#define __STDC_LIMIT_MACROS
```

Comment:
I wonder if it would be slightly safer to put this before *all* includes. For example, this would fail if stdlib.h included stdint.h, because by this point it would be too late to `#define __STDC_LIMIT_MACROS`.

Also a comment mentioning why we need it would be helpful. Something like:

```
#define __STDC_LIMIT_MACROS  // To make stdint.h define UINT64_MAX
```